bpf/lib/egress_policies.h

```diff
@@ -38,10 +43,9 @@ bool egress_gw_request_needs_redirect(struct iphdr *ip4, __u32 *tunnel_endpoint)
 	if (!egress_gw_policy)
 		return false;
 
-	/* If the gateway IP is 0.0.0.0 it means this is an excluded CIDR, so
-	 * skip redirection
-	 */
-	if (!egress_gw_policy->gateway_ip)
+	/* FIXME: no gateway traffic should be dropped in a future release */
+	if (egress_gw_policy->gateway_ip == EGRESS_GATEWAY_NO_GATEWAY ||
+            egress_gw_policy->gateway_ip == EGRESS_GATEWAY_EXCLUDED_CIDR)
```

Comment:
BPF checkpatch is complaining about this:
```
   Error: ERROR:CODE_INDENT: code indent should use tabs where possible
  #44: FILE: bpf/lib/egress_policies.h:48:
  +            egress_gw_policy->gateway_ip == EGRESS_GATEWAY_EXCLUDED_CIDR)$
  
  CHECK:PARENTHESIS_ALIGNMENT: Alignment should match open parenthesis
  #44: FILE: bpf/lib/egress_policies.h:48:
  +	if (egress_gw_policy->gateway_ip == EGRESS_GATEWAY_NO_GATEWAY ||
  +            egress_gw_policy->gateway_ip == EGRESS_GATEWAY_EXCLUDED_CIDR)
```

but let's wait until the tests finish running before pushing again